test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -149,7 +149,7 @@ public static TheoryData ResolveDirectiveDescriptorsInvalidTagHelperPrefixData
                             },
                             new TagHelperDirectiveDescriptor
                             {
-                                DirectiveText = "*Plain*, " + assemblyA,
+                                DirectiveText = "Microsoft.AspNet.Razor.Runtime.TagHelpers.Valid_P*, " + assemblyA,
```

Comment:
Do these set of tests (`*___*`) do anything unique compared to other variations here. Aka do we have other tests that appropriately cover `___*`?